aws_jobs/cyhy-data-extract.py

```diff
@@ -138,12 +138,13 @@ def cleanup_bucket_files(object_retention_days):
                 ]
             },
         )
-        for err in del_resp["Errors"]:
-            sys.stderr.write(
-                "Error: {} when deleting {} - {}\n".format(
-                    err["Message"], err["Key"], err["Code"]
+        if "Errors" in del_resp:
```

Comment:
You can just use del_resp.get("Errors") instead.   This is simpler.   

```
Help on built-in function get:

get(key, default=None, /) method of builtins.dict instance
    Return the value for key if key is in the dictionary, else default.
(END)
```

Reply:
Sorry I thought the if would be a bit more blatant about what was being done if someone was skimming. Based on feedback both you and @dav3r I have changed to a get() with default value.